### DISCUSSION TITLE
Separate Schema Files Per Model

### DISCUSSION THREAD

**Discussion by Skillz4Killz** [prisma/prisma#3185] [Q&A] [answered]
Simple but silly question I was not able to find in the docs. Is it possible to have separate files per model?

**Comment by ryands17** [comment, marked as answer]
Hey @Skillz4Killz 👋 
Currently it is not possible to have separate files per model and there is an open issue [here](https://github.com/prisma/prisma/issues/2377) for that as well. Do add a 👍 on this so that we can infer the priority of this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
